src/util.cpp

```diff
@@ -801,11 +801,9 @@ int GetNumCores()
 
 std::string CopyrightHolders(const std::string& strPrefix)
 {
-    std::string strCopyrightHolders = strPrefix + _(COPYRIGHT_HOLDERS);
-    if (strCopyrightHolders.find("%s") != strCopyrightHolders.npos) {
-        strCopyrightHolders = strprintf(strCopyrightHolders, _(COPYRIGHT_HOLDERS_SUBSTITUTION));
-    }
-    if (strCopyrightHolders.find("Bitcoin Core developers") == strCopyrightHolders.npos) {
+    std::string strCopyrightHolders = strPrefix + strprintf(_(COPYRIGHT_HOLDERS), _(COPYRIGHT_HOLDERS_SUBSTITUTION));
```

Comment:
> Adding some heuristics to detect such additional string in some downstream helper method is good intention but doesn't really help, imo. Add a line note

The point here was to make it impossible to get rid of our copyright by just changing a build system variable. It feels incredibly brittle if all of the copyright string hinges on is something that is passed to the compiler as `-D` define.

Agree with adding documentation of course, but this heuristic should stay.